backend/lib/endpoints/posts.js

```diff
@@ -321,7 +327,7 @@ async function routes(app) {
         throw app.httpErrors.internalServerError();
       } else if (post === null) {
         throw app.httpErrors.notFound();
-      } else if (!userId.equals(post.author.id)) {
+      } else if (userId !== post.author.id.toString()) {
```

Comment:
Why change this to use string comparison instead of the ObjectId equals?

Reply:
I had to change it to this because I wasn't getting the signup email to validate my account. It was an issue with my .env configs.

I'll change it back!  Thanks for bringing this to my attention. This is still wip.